prisma/seed.ts

```diff
@@ -0,0 +1,54 @@
+import { PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
```

Comment:
Let's try to use functions from our shared db so we avoid creating new PrismaClients in multiple places